VivistaServer/wwwroot/style.css

```diff
@@ -735,6 +735,40 @@ input.toggle + label:active::after {
 	visibility: hidden;
 }
 
+.chartBox {
+	width: 50%;
+	min-height: 500px;
+	min-width: 500px;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.table-center {
+	margin-left: auto;
+	margin-right: auto;
+}
+
+	.table-center td{
+		text-align: center;
+		vertical-align: central;
+	}
+	.table-center th {
+		text-align: center;
+		vertical-align: central;
+	}
+
+
+.center-text {
+	text-align: center;
+}
+
+.display-center {
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+

```

Comment:
Zulke "algemene" CSS regels staan bovenaan, rond lijn 90

Reply:
Dat met die .Center heb ik geprobeerd, maar dat lukt niet bij alles daarom dat ik zo moet oplossen.

Reply:
Ik bedoel dat je deze code naar boven moet verplaatsen, zodat het bij de rest van de "algemene" regels staat.

Reply:
Aah oke cava komt in orde